### ISSUE TITLE
[site request] bang.com

### ISSUE THREAD

**Issue by PAHelper2** [PAhelper/PhoenixAdult.bundle#57]
Would be great to support bang.com.

**Comment by PAhelper** [comment]
I agree, as more sites get added, that one gets closer to the top of my list. It has a decent search on their site iirc, so should be easy to implement. I've already got this in #1 so I'm going to close this issue, but rest assured it's coming.
